packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -368,12 +380,12 @@ describe('RPC Server Acceptance Tests', function() {
 
     it('should execute "eth_getBalance" for contract', async function() {
         const res = await relay.call('eth_getBalance', [Utils.idToEvmAddress(contractId.toString()), 'latest']);
-        expect(res).to.eq('0x56bc75e2d63100000');
+        expect(res).to.eq('0xde0b6b3a7640000');
```

Comment:
I think that it would be better to introduce var at the top of the file
```TypeScript
const ONE_WEIBAR = ethers.utils.parseEther('1', 18)
```
and here assert:
```TypeScript
expect(res).to.eq(ethers.utils.hexValue(ONE_WEIBAR));
```